idunn/directions/client.py

```diff
@@ -159,7 +159,15 @@ def get_directions(self, from_loc, to_loc, mode, lang, **extra):
         else:
             raise BadRequest(f'unknown mode {mode}')
 
-        return method(from_loc, to_loc, mode, lang, **extra)
+        method_name = method.__name__
```

Comment:
Why creating a temp var?

Reply:
Because it's used twice in the following call, and it makes easier to replace it with a different name.  
Should I change it ?

Reply:
Nope, all good!